src/PraxisNote.Web/ClientApp/src/app/meetings/audio-recorder.service.ts

```
@@ -0,0 +1,225 @@
+    if (this.levelAnimationId !== null) {
+      cancelAnimationFrame(this.levelAnimationId);
+      this.levelAnimationId = null;
+    }
+  }
+
+  private cleanup(): void {
+    this.stopTimer();
+    this.stopLevelMetering();
+
+    if (this.audioStream) {
+      this.audioStream.getTracks().forEach(track => track.stop());
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)